crates/hir_def/src/nameres/tests/macros.rs

```diff
@@ -677,7 +677,7 @@ fn macro_expansion_overflow() {
         r#"
 macro_rules! a {
     ($e:expr; $($t:tt)*) => {
-        b!($($t)*);
```

Comment:
The old test actually is almost valid, it was not working because of the bug in #5896

[playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=2820790f7dfd3a70ead75d60d067145a)